### PR TITLE
RN 34857

### PR DESCRIPTION
**Pull request by EdithLansens**

### CODE REVIEW

File: release-notes/Cube/Cube_Feature_Release_10.3/Cube_10.3.1.md [hunk 1]
```diff
@@ -26,6 +26,14 @@ This setting allows you to specify the maximum amount of memory that SLAnalytics
 
 Default value: 2.00 GB
 
+#### DataMiner Cube - Visual Overview: Session variable YAxisResources now supports filters [ID_34857]
+
+<!-- MR 10.4.0 - FR 10.3.1 -->
+
+From now on, you can pass filters to the YAxisResources session variable in order to show the corresponding resource bands.
+
+The number of resources shown on a timeline is limited to 100 resources to avoid delay while loading the bands.
```
Comment (HansSolos) [inline release-notes/Cube/Cube_Feature_Release_10.3/Cube_10.3.1.md:35]: There is no software limit of 100 resources.
Better would be to say that is recommended to use a filter which results in less than 100 resources.

File: user-guide/Basic_Functionality/Visio/SRM/Embedding_a_Resource_Manager_component.md [hunk 2]
```diff
@@ -340,6 +340,15 @@ To also show resources for elements in child views, in the **ComponentOptions**
...
 > The corresponding resource bands are not updated automatically in case there is a change to the configuration of the elements.
 
+#### Passing exposers
+
+From DataMiner 10.3.1/10.4.0 onwards, you can pass exposers by object ID.
+
+This enables a new filter option that can be used to show the corresponding resource bands on the timeline.
+
+> [!NOTE]
+> The number of resources shown on a timeline is limited to 100 resources to avoid delay while loading the bands.
```
Comment (HansSolos) [inline user-guide/Basic_Functionality/Visio/SRM/Embedding_a_Resource_Manager_component.md:350]: As stated above, the warning in the RN is not a software limit, but a warning to the user for using the filter wisely.

File: user-guide/Basic_Functionality/Visio/SRM/Embedding_a_Resource_Manager_component.md [hunk 3]
```diff
@@ -340,6 +340,15 @@ To also show resources for elements in child views, in the **ComponentOptions**
 > [!NOTE]
 > The corresponding resource bands are not updated automatically in case there is a change to the configuration of the elements.
 
+#### Passing exposers
+
+From DataMiner 10.3.1/10.4.0 onwards, you can pass exposers by object ID.
```
Comment (HansSolos) [inline user-guide/Basic_Functionality/Visio/SRM/Embedding_a_Resource_Manager_component.md:345]: should change to "you can pass resource exposer as a filter"

File: user-guide/Basic_Functionality/Visio/SRM/Embedding_a_Resource_Manager_component.md [hunk 4]
```diff
@@ -340,6 +340,15 @@ To also show resources for elements in child views, in the **ComponentOptions**
...
+
+This enables a new filter option that can be used to show the corresponding resource bands on the timeline.
+
```
Comment (HansSolos) [inline user-guide/Basic_Functionality/Visio/SRM/Embedding_a_Resource_Manager_component.md:348]: I would add quite some examples, as stated in internal comments in the RN.

**Review by HansSolos — CHANGES_REQUESTED** [review]

**Comment by EdithLansens** [comment]
Could you take another look, @HansSolos? Thanks!

File: user-guide/Basic_Functionality/Visio/SRM/Embedding_a_Resource_Manager_component.md [hunk 5]
```diff
@@ -340,14 +340,22 @@ To also show resources for elements in child views, in the **ComponentOptions**
...
+This enables a new filter option that can be used to show the corresponding resource bands on the timeline. Using a filter which results in less than 100 resources is recommended to avoid delay while loading the bands.
+
+> [!TIP]
+> See also: [Filter feature examples](xref:YAxisResource_Shape_Data_Examples#examples-filter-feature)
+
+A converter has been created in the client which will convert a profile parameter name to the ID of the found object. For this, the format [ProfileParameter:xxx,ID] should be used, replacing 'xxx' with the name of the capacity/capability profile parameter.
 
 > [!NOTE]
-> The number of resources shown on a timeline is limited to 100 resources to avoid delay while loading the bands.
+> The server side filter is case sensitive with regard to profile parameter keys.
```
Comment (HansSolos) [inline user-guide/Basic_Functionality/Visio/SRM/Embedding_a_Resource_Manager_component.md:355]: Could you also add that it is expected that the profile parameter keys are represented with the number format.
Example:
- {d248b04f-2253-41c6-a92a-e12768fe2b20} is incorrect, this is format B (braces)
- (d248b04f-2253-41c6-a92a-e12768fe2b20) is incorrect, this is format P (parentheses)
- d248b04f-2253-41c6-a92a-e12768fe2b20 is correct, this is format D (default)
- d248b04f225341c6a92ae12768fe2b20 is correct, this is format N (number)

more info: https://learn.microsoft.com/en-us/dotnet/api/system.guid.tostring?view=netframework-4.8

**Review by HansSolos — CHANGES_REQUESTED** [review]

**Review by HansSolos — APPROVED** [review]

**Pull request merged**